lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -130,6 +130,12 @@ <h3 class="hd hd-3">${_("Regenerate Certificates")}</h3>
                 </div>
                 <div>
                     <label>
+                        <input id="certificate_status_verified_users_with_audit_certs" type="checkbox" name="verified_with_audit_certs" value="verified_users_with_audit_certs">
```

Comment:
why aren't we pulling the value from `section_data['status']`, like we're doing for the other inputs?